mkdocs.yml

```diff
@@ -29,44 +33,68 @@ theme:
-        - levenscyclus/monitoring-en-beheer.md
-        - levenscyclus/uitfaseren.md
-    - Onderwerpen:
-        - onderwerpen/index.md
-        - ... | onderwerpen/*/*.md
+        - Overzicht: levenscyclus/index.md
+        - Probleemanalyse: levenscyclus/probleemanalyse.md
+        - Ontwerp: levenscyclus/ontwerp.md
+        - Dataverkenning en Datapreparatie: levenscyclus/dataverkenning-en-datapreparatie.md
+        - Ontwikkelen: levenscyclus/ontwikkelen.md
+        - Verificatie en Validatie: levenscyclus/verificatie-en-validatie.md
+        - Implementatie: levenscyclus/implementatie.md
+        - Monitoring en Beheer: levenscyclus/monitoring-en-beheer.md
+        - Uitfaseren: levenscyclus/uitfaseren.md
+    - Onderwerpen: 
+        - Overzicht: onderwerpen/index.md
+        - Bias en non discriminatie: onderwerpen/bias-en-non-discriminatie/index.md
+        - Data: onderwerpen/data/index.md
+        - Duurzaamheid: onderwerpen/duurzaamheid/index.md
+        - Fundamentele rechten: onderwerpen/fundamentele-rechten/index.md
+        - Mensenlijke controle: onderwerpen/menselijke-controle/index.md
+        - Privacy en gegevensbescherming: onderwerpen/privacy-en-gegevensbescherming/index.md
+        - Publieke inkoop: onderwerpen/publieke-inkoop/index.md
+        - Technische robuustheid en veiligheid: onderwerpen/technische-robuustheid-en-veiligheid/index.md
+        - Transparantie: onderwerpen/transparantie/index.md
+        - Transparantie: bouwblokken/transparantie/index.md
     - Rollen:
-        - ... | rollen/*.md
-    - Vereisten:
-        - ... | vereisten/*.md
-    - Maatregelen:
-        - ... | maatregelen/*.md
-    - Instrumenten:
-        - ... | instrumenten/*.md
+        - Overzicht: rollen/index.md
+        - Aanbestedingsjurist: rollen/aanbestedingsjurist.md
+        - Aanbieder: rollen/aanbieder.md
+        - Archiefdeskundige: rollen/archiefdeskundige.md
+        - Behoeftesteller: rollen/behoeftesteller.md
+        - Beleidsmedewerker: rollen/beleidsmedewerker.md
+        - Communicatieadviseur: rollen/communicatieadviseur.md
+        - Contractbeheerder: rollen/contractbeheerder.md
+        - Data-engineer: rollen/data-engineer.md
+        - Data-scientist: rollen/data-scientist.md
+        - Domeinspecialist: rollen/domeinspecialist.md
+        - Ethicus: rollen/ethicus.md
+        - Gebruiker: rollen/gebruiker.md
+        - Gemandateerd-verantwoordelijke: rollen/gemandateerd-verantwoordelijke.md
+        - Informatie-analist: rollen/informatie-analist.md
+        - Informatiebeheerder: rollen/informatiebeheerder.md
+        - Inkoopadviseur: rollen/inkoopadviseur.md
+        - Jurist: rollen/jurist.md
+        - Opdrachtgever: rollen/opdrachtgever.md
+        - Opdrachtnemer: rollen/opdrachtnemer.md
+        - Privacy-officer: rollen/privacy-officer.md
+        - Proceseigenaar: rollen/proceseigenaar.md
+        - Projectleider: rollen/projectleider.md
+        - Security-officer: rollen/security-officer.md
```

Comment:
Dit zijn er een beetje te veel denk ik. Hier moeten we in snoeien en ze een beetje groeperen.